docs/components/switch/index.html

```diff
@@ -0,0 +1,98 @@
+---
+title: Switch
+minver: 0.22.0
+also:
+components/switch/test.html: Testing - Switch
+elements/hx-switch: <hx-switch>
+  ---
+  {% extends 'component.njk' %}
+
+  {% block page_header %}
+  {# TODO: add component description #}
+  {% endblock %}
+
+  {% block content %}
+  <section>
+    <header>
+      <h2 id="basic-checkbox">Switch</h2>
+      <p>A switch is used when a user needs to make a binary decision, such as turning a setting on or off or setting a preference enabling or disabling a device or system</p>
```

Comment:
Please move this line up to the page header:

```
{% block page_header %}
  // INSERT HERE 
{% endblock %}
```

Also, add line breaks at around 80 characters, depending on the word.  If the word goes beyond 80 characters, truncate the line to the previous word.

Reply:
Changes have been done, please review.